src/doc/about.rst

```diff
@@ -0,0 +1,156 @@
+      a new command-line decoder (that calls the same underlying templated code).
+- Kaldi code is easy to understand.
+    - Even though the Kaldi toolkit as a whole may get very large, we aim
+      for each individual part of it to be understandable without too much
+      effort.  We will accept some code duplication if it improves the
+      understandability of individual pieces.
+- Kaldi code is easy to reuse and refactor.
+    - We aim for the toolkit to as loosely coupled as possible.
+      In general this means that any given header should need to \#include as
+      few other header files as possible.  The matrix library, in particular,
+      only depends on code in one other subdirectory so it can be used independently
+      of almost all the rest of Kaldi.
+
+---------------------
+Status of the project
+---------------------
+  
+Currently, we have code and scripts for most standard techniques, including all standard
+linear transforms, MMI, boosted MMI and MCE discriminative training, and also feature-space
+discriminative training (like fMPE, but based on boosted MMI).
+We have working recipes for Wall Street Journalcand Resource Management, 
+and also for Switchboard.   The Switchboard recipe is not yet
+giving state-of-the-art results, due to vocabulary and language model issues-- we
+don't use any external data sources for this.
+
+Note: after an early phase in which we intended to use version numbers for
+major releases of Kaldi ("v1" and so on), we realized that these type of
+releases do not mesh well with the natural style of development, which is very
+continuous.  Currently we maintain only the "master" development branch, and
+this is the version you should use. Also,
+frequently do "git pull" to keep it up to date; see \ref install for more details.
+
+---------------------------
+Referencing Kaldi in papers
+---------------------------
+
+You can use the following reference if you want to cite Kaldi in papers.
+
+.. code-block:: bibtex
+
+    @INPROCEEDINGS{
+             Povey_ASRU2011,
+             author = {Povey, Daniel and Ghoshal, Arnab and Boulianne, Gilles and Burget, Lukas and Glembek, Ondrej and Goel, Nagendra and Hannemann, Mirko and Motlicek, Petr and Qian, Yanmin and Schwarz, Petr and Silovsky, Jan and Stemmer, Georg and Vesely, Karel},
+           keywords = {ASR, Automatic Speech Recognition, GMM, HTK, SGMM},
+              month = dec,
+              title = {The Kaldi Speech Recognition Toolkit},
+          booktitle = {IEEE 2011 Workshop on Automatic Speech Recognition and Understanding},
+               year = {2011},
+          publisher = {IEEE Signal Processing Society},
+           location = {Hilton Waikoloa Village, Big Island, Hawaii, US},
+               note = {IEEE Catalog No.: CFP11SRW-USB},
+    }
+
+The paper can be found <a href=http://publications.idiap.ch/downloads/papers/2012/Povey_ASRU2011_2011.pdf> here </a>.
```

Comment:
The paper can be found
`here <http://publications.idiap.ch/downloads/papers/2012/Povey_ASRU2011_2011.pdf>`_

<a/> is not valid in rst.